.cargo/config

```diff
@@ -1,6 +1,5 @@
 [build]
 target-dir = "build/cargo_target"
-target = "x86_64-unknown-linux-musl"
```

Comment:
As part of testing can we run this in both ARM/x86? https://github.com/firecracker-microvm/firecracker/blob/main/.buildkite/pipeline_pr.py#L52

Reply:
we could use `file` which is already available in the devctr :
```
root:/firecracker# file ./build/cargo_target/aarch64-unknown-linux-musl/debug/firecracker | grep "statically linked"
./build/cargo_target/aarch64-unknown-linux-musl/debug/firecracker: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, BuildID[sha1]=f401b1c0598407f0a0ca9027c61b01fe1b699d23, with debug_info, not stripped
```